app/lib/logic/provider_upgrade.rb

```diff
@@ -61,7 +61,7 @@ def hideable_switches
         # In https://github.com/3scale/system/pull/9326, this method changes to `settings.switches.select { |_name, switch| switch.hideable? }`,
         # intending to fix this problem in the next PR.
         #
-        settings.switches.select { |_name, switch| switch.hideable? }
+        settings.switches.hideable
```

Comment:
Good, but since we changed this again, we should probably make a Jira issue explaining all what we have commented above in this method because otherwise it will never be addressed 😝